webui/src/components/dashboard/PanelChart.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <q-card flat bordered>
+    <q-card-section>
+      <div class="row items-center no-wrap">
+        <div class="col">
+          <div class="text-h6 text-weight-bold">{{getName()}}</div>
+        </div>
+        <!--div class="col-auto">
+          <q-btn :to="getUrl()" color="accent" dense flat icon-right="eva-arrow-forward-outline" no-caps label="Explore" size="md" class="text-weight-bold"/>
+        </div-->
```

Comment:
could you remove this comment block.

Reply:
done:
https://github.com/containous/traefik/pull/5249/commits/13c4fadce79822dbe8f77eb51dfef6685b8ff509